src/client/components/forms/userCollection.js

```diff
@@ -262,21 +262,25 @@ class UserCollectionForm extends React.Component {
 							<div className={errorAlertClass}>
 								<Alert bsStyle="danger">Error: {errorText}</Alert>
 							</div>
-							<div className="text-center">
-								<Button
-									bsStyle="primary"
-									type="button"
-									onClick={this.handleAddCollaborator}
-								>
-									<FontAwesomeIcon icon="plus"/>
-									&nbsp;Add another collaborator
-								</Button>
-								<Button
-									bsStyle="success"
-									type="submit"
-								>
-									<FontAwesomeIcon icon="save"/>&nbsp;{submitLabel}
-								</Button>
+							<div className="row">
+								<div className="col-sm-6 margin-top-d5">
+									<Button
+										bsStyle="primary"
+										type="button"
+										onClick={this.handleAddCollaborator}
+									>
+										<FontAwesomeIcon icon="plus"/>
+										&nbsp;Add another collaborator
+									</Button>
+								</div>
+								<div className="col-sm-6 margin-top-d5">
+									<Button
+										bsStyle="success"
+										type="submit"
+									>
+										<FontAwesomeIcon icon="save"/>&nbsp;{submitLabel}
+									</Button>
+								</div>
 								{
 									this.props.collection.id ?
 										<Button
```

Comment:
I just noticed the third button ("Delete"), which is only visible when you edit one of your own collections.

It should receive the same changes as the other buttons, meaning it should also be wrapped in a `<div>` in the same way and the button should have the `block` attribute.

Here's the full block of code:

```jsx
{
	this.props.collection.id ?
		<div className="col-sm-6 margin-top-d5">
			<Button
				block
				bsStyle="danger"
				type="button"
				onClick={this.handleShowModal}
			>
				<FontAwesomeIcon icon="trash-alt"/>&nbsp;Delete collection
			</Button>
		</div> : null
}
```

With three buttons in mind, it's worth you having another pass to see if it looks good with the three buttons.